authority.go

```diff
@@ -0,0 +1,11 @@
+package versionbundle
+
+import (
+	"net/url"
+)
+
+type Authority struct {
+	Endpoint *url.URL `yaml:"endpoint"`
+	Name     string   `yaml:"Name"`
+	Version  string   `yaml:"version"`
```

Comment:
`Provider` needs to be added here.

Reply:
Why do we need to add a provider?

Reply:
I thought there could be a map having the provider as index. I do not like to hard code the provider somehow. When you think this is right then lets move on but I feel like it is wrong.

Reply:
I would rather try to find a way to get rid of the whole field, but I don't have working idea for `cluster-operator` yet.